CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+*Megaparsec follows [SemVer](https://semver.org/).*
```

Comment:
Copying is okay, but I think the same of the project should be changed to match :)

Reply:
Oops. Done!